record-and-playback/core/lib/recordandplayback/edl/media_utils.rb

```
@@ -0,0 +1,102 @@
+
+        BigBlueButton.logger.debug("Getting #{stream_type} packet PTS times for #{File.basename(filename).inspect}: #{Shellwords.join(ffprobe_cmd)}")
+
+        pts_gaps = []
+        Tempfile.create(['ffprobe', '.log'], process_dir) do |log_file|
+          IO.pipe do |r, w|
+            pid = spawn(
+              *ffprobe_cmd,
+              close_others: true,
+              in: :close,
+              out: w,
+              err: %i[child out],
```

RuboCop (1.85.1)

[convention] Avoid comma after the last parameter of a method call.

(Style/TrailingCommaInArguments)